packages/cli/src/info/indent.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Adds two spaces of indentation to the given line.
+ * @param line The line to be indented.
+ * @returns The indented line.
+ */
+export const indent = (line: string) => `  ${line}`;
```

Comment:
this should be printer functionality.

Reply:
This could also be another argument for the "format" object I suggested [here](https://github.com/aws-amplify/amplify-backend/pull/905#discussion_r1449321497). Then usage would be something like `printer.print(format.indent('stuff'))`